docs/guide/getting-started/README.md

```diff
@@ -77,6 +77,7 @@ services:
     image: koenkk/zigbee2mqtt
     volumes:
       - ./zigbee2mqtt-data:/app/data
+      -v /run/udev:/run/udev:ro
```

Comment:
`-v` looks like a typo ( should be  `-` i think)

Reply:
thx, force pushed it :)